.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,25 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "1.20"
```

是說用單引號 484 好一點，有需要再使用雙引號

```suggestion
          go-version: '1.20'
```

---

好的，這份是從它官網抄過來沒有經過調整～

---

Yaml 字串預設是雙引號，VSCode formatter 會自動把它變成雙引號